packages/filecoin-rpc-client/src/index.ts

```diff
@@ -1,64 +1,50 @@
-import axios, { AxiosRequestConfig } from 'axios'
+import axios, { AxiosRequestConfig, AxiosRequestHeaders } from 'axios'
 
-type Headers = Record<string, string | null | undefined>
+export const removeEmptyHeaders = (
+  headers: AxiosRequestHeaders
+): AxiosRequestHeaders =>
+  Object.fromEntries(Object.entries(headers).filter(([_key, value]) => !!value))
 
-export function removeEmptyHeaders(headers: Headers): Record<string, string> {
-  const newHeaders: Record<string, string> = {}
-  Object.keys(headers).forEach((key) => {
-    if (headers[key]) newHeaders[key] = headers[key] as string
-  })
-  return newHeaders
-}
```

Comment:
I've updated this a bit, but I'm not sure why we need to check for empty headers in the first place. The axios options expect the headers to be `Record<string, string>`, so apart from an empty string, it's not possible to enter any falsy values (when using TypeScript). But I don't see why a user / app _would_ enter an empty string or falsy value anyway.

Reply:
Early early on when using this package we had to _sometimes_ use a `token`, so it made it easier to have the `token` value be falsey. The changes you made seem to work well